notebooks/compute-network-v2.ipynb

```diff
@@ -0,0 +1,464 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "import sys\n",
+    "import subprocess\n",
+    "\n",
+    "# logic to accomodate Google Colab\n",
+    "try:\n",
+    "    import google.colab\n",
+    "\n",
+    "    ENV_IS_CL = True\n",
+    "    \n",
+    "    root = r\"/content/t-route\"\n",
+    "    subprocess.run([\"git\", \"clone\", \"https://github.com/NOAA-OWP/t-route.git\"])\n",
+    "    sys.path.append(\"/content/t-route/src/python_framework_v02\")\n",
+    "    sys.path.append(\"/content/t-route/src/python_routing_v02\")\n",
+    "    sys.path.append(\"/content/t-route/src/python_framework_v01\")\n",
+    "    \n",
+    "    fortran_source_dir = os.path.join(root, \"src\", \"fortran_routing\", \"mc_pylink_v00\", \"MC_singleSeg_singleTS\")\n",
+    "    sys.path.append(fortran_source_dir)\n",
+    "\n",
+    "    ! pip install geopandas\n",
+    "    ! pip install netcdf4\n",
```

Comment:
If we use `subprocess.run(["pip", "install","geopandas"])` and `subprocess.run(["pip", "install","netcdf4"])` then the nbformat.py script will not complain about the `!` syntax -- as it stands, it is probably just skipping this cell.

Reply:
👍